apps/arena/priv/config.json

```diff
@@ -491,6 +495,26 @@
           "effect_delay_ms": 400
         }
       }
+    },
+    {
+      "name": "muflus_death",
+      "duration_ms": 5000,
+      "remove_on_action": false,
+      "effect_mechanics": {
+        "damage_up": {
+          "amount": 30
+        }
+      }
+    },
+    {
+      "name": "h4ck_death",
+      "duration_ms": 5000,
+      "remove_on_action": false,
```

Comment:
These effects should be called speed and damage. I know they belong to muflus and h4ck, and they trigger on death, but that may change in the future and could be reusable like the invisible one.

Reply:
That makes sense until we have 2 speed (or whatever) effects with slightly different configs. Also, the client probably uses the name of the effect for the visual indicators

PS: once we move items to be effect we do have 2 speed effects